.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           RELEASE_SSH_KEY: ${{ secrets.RELEASE_SSH_KEY }}
           SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN }}
         # Put your action repo here
-        uses: SonarSource/gh-action_release/main@v4
+        uses: SonarSource/gh-action_release/main@4.0.1
```

Comment:
I prefer the `@v4` here to be on the latest version from REs. Otherwise, we would stay behind and might get surprises. It is the strategy suggested by Tom in his email, where he provided this as a template
https://github.com/SonarSource/sonar-dummy-oss/blob/master/.github/workflows/release.yml

It is also what Evgeny did for every other repo in LT.

Reply:
Actually, I've checked with Tom the usage of tags and this was his reply:

> it is indeed a safer approach, maybe we should recommend it as well, I'll check witht the team

Reply:
I'd rather play simple here, to get `@v4` now and update to `v4.0.1` once RE team finishes their discussion. We don't know what the outcome will be.

Reply:
I'm thinking the other way around. I would like to have the versions fixed to avoid any further surprises. Referencing a branch means the code might change from one release to another and this could theoretically break our build. Figuring out what changed and where could be tricky.

If there will be a reason for us to update, they will let us know and we will do it. If not, we can periodically check and change the linked versions. 

Why would they use releases and not use them?